frontend/pages/ManageControlsPage/OSSettings/cards/DiskEncryption/components/DiskEncryptionTable/DiskEncryptionTableConfig.tsx

```diff
@@ -94,10 +93,7 @@ const defaultTableHeaders: IDataColumn[] = [
       return (
         <div className="disk-encryption-table__aggregate-table-data">
           <TextCell value={aggregateCount} formatter={(val) => <>{val}</>} />
-          {/* TODO: WINDOWS FEATURE FLAG: remove this conditional when windows mdm
-          is released. the view all UI will show in the windows column when we
-          release the feature. */}
-          {!original.includeWindows && (
+          {!true && (
```

Comment:
So here I'm really not sure what to do? By removing `mdm_enabled`, `includeWindows` is now always `true`, so that means I should delete the whole `{...}` part? So `<ViewAllHostsLink>` would not be rendered anymore, does that sound correct?

Reply:
We want View all hosts to always appear now so I'm gonna remove the conditional and just render that component